domain-models-runtime/src/main/java/org/folio/rest/persist/PostgresTesterEmbedded.java

```diff
@@ -0,0 +1,66 @@
+package org.folio.rest.persist;
+
+import org.apache.logging.log4j.LogManager;
+import org.apache.logging.log4j.Logger;
+import org.folio.rest.tools.utils.NetworkUtils;
+import org.folio.util.PostgresTester;
+import ru.yandex.qatools.embed.postgresql.EmbeddedPostgres;
+import ru.yandex.qatools.embed.postgresql.distribution.Version;
+
+import java.io.IOException;
+import java.io.UncheckedIOException;
+import java.util.Arrays;
+
+public class PostgresTesterEmbedded implements PostgresTester {
```

Comment:
This class should be deprecated. The deprecation note should link to the replacement (PostgresTesterContainer) and mention that this class will be removed in the next major RMB version.

Reply:
Yes. Will fix that.

Reply:
Done now. I also made it package private.. I don't see why anybody would use it!!

Reply:
Done now. I also made it package private.. I don't see why anybody would use it!!